pages/osx/say.md

```diff
@@ -15,10 +15,14 @@
 
 `say --voice={{voice}} --rate={{words_per_minute}} "{{I'm sorry Dave, I can't let you do that.}}"`
 
-- List the available voices:
+- List the available voices, different voices speak in different languages:
```

Comment:
Wouldn't `List the available voices (different voices speak in different languages):` be prettier and in accordance with other pages?

Reply:
Or 
```suggestion
- List the available voices with different voices which speak in different languages:
```

Reply:
I think @yutyo's suggestion is better, updating.